File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioSection.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnThemeDirection;
import com.neome.java.api.meta.base.Types.EnumDefnThemeSectionVariant;
import com.neome.java.api.meta.base.Types.MetaIdSection;
import com.neome.java.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioSection extends StudioComposite
{
  @Nullable
  public EnumDefnThemeDirection direction;

  public MetaIdSection metaId;

  @Nullable
  public MetaIdVar propertyEditorLabelVarId;

  @Nullable
  public EnumDefnThemeSectionVariant sectionVariant;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioSection.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnThemeDirection
import com.neome.api.meta.base.Types.EnumDefnThemeSectionVariant
import com.neome.api.meta.base.Types.MetaIdSection
import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.meta.base.dto.StudioComposite

open class StudioSection : StudioComposite()
{
  var direction: EnumDefnThemeDirection? = null
  lateinit var metaId: MetaIdSection
  var propertyEditorLabelVarId: MetaIdVar? = null
  var sectionVariant: EnumDefnThemeSectionVariant? = null
}